README.md

```diff
@@ -1,2 +1,30 @@
 # derp
-A first attempt at a simple LSST DRP catalog emulator
+
+A first attempt at a simple LSST DRP catalog emulator. 
+
+A number of planned DESC project will either need, or could use, a wide area mock LSST catalog, including stars, galaxies, and artifacts, with realistic errors and uncertainties, and with the correct anticipated schema, as would be generated by the LSST Facility's Data Release Production (DRP) pipeline and released to the LSST science community. Most applications will need a mock Object table, while some will need mock ForcedDIASource light curves. DC2 will provide a 300 square degree DRP catalog that we can use to train an emulator, that can then take in a much larger "truth" catalog and generate the corresponding wide area mock LSST catalog.
+
+## Limitations
+
+What makes `derp` so simple?
+* We ignore all blended objects.
+* We only emulate the DRP Object table, so Sources or DIASources.
+* The code only runs at NERSC, where the DC2 data live. We are targeting the DC2 Run 1.1p Truth table and DRP Object table.
+* We plan a naive application of a generative machine learning model, to map N truth properties to M observed properties.
+
+## Sources
+
+This project builds on:
+- Sean Mclaughlin's CMU 2018 Hack Day project [`blendtec`](https://github.com/mclaughlin6464/blendtec)
+- The [DC2-analysis tutorial notebooks](https://github.com/LSSTDESC/DC2-analysis/blob/master/tutorials/README.rst)
+
+## Contact
+
+Comments, questions, feedback etc welcome, via the [issues](https://github.com/LSSTDESC/derp/issues/new?body=@drphilmarshall).
```

Comment:
I did not know this trick for getting it to start a new issue addressed at you. Very cool.

Reply:
Thanks! I'm preparing for a time when I am _not_ reading all LSST DESC GitHub notifications ;-)

Reply:
Oh no, will you go into withdrawal??